indexer.py

```diff
@@ -337,6 +344,40 @@ def new(self, session, indx, block, args):
 
             ProtocolCoverage.update(session, protocol.id, coverage_amount, timestamp)
 
+    class ClaimCreated:
+        def new(self, session, indx, block, args):
+            protocol_bytes_id = Protocol.parse_bytes_id(args["protocol"])
+            protocol = Protocol.get(session, protocol_bytes_id)
+
+            if not protocol:
```

Comment:
We should log an error here, it's critical if a `ClaimCreated` event comes in and the protocol is not in the database.